.buildkite/steps/build-lambda.sh

```diff
@@ -2,14 +2,15 @@
 set -eu
 
 docker run --rm --volume "$PWD:/code" \
-  --workdir "/code" \
+  --workdir /code \
   --rm \
-    golang:1.20 \
-    sh -c "go get ./lambda && go build -o ./lambda/handler ./lambda"
+  --env CGO_ENABLED=0 \
+  golang:1.20 \
```

Comment:
Should we update the lambda compile container to 1.21?

Reply:
Yeah, I missed that. Thanks for bring it up.